CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 1.4.0 (February 28, 2024)
```

Comment:
Assume we'll just regenerate this later by re-running the make target (if backports happen)?

Reply:
Yeah, that's a good call. TBH I totally forgot about it reading the instructions. I'm inclined to say keeping it is harmless if we don't have changes but open to feedback.